package.json

```diff
@@ -22,7 +22,7 @@
     "gulp": "gulp",
     "karma": "karma",
     "karma.start": "karma start",
-    "postinstall": "typings install && gulp check.versions && npm prune && gulp webdriver && node tools/install.js",
+    "postinstall": "typings install && gulp check.versions && npm prune && gulp webdriver && cd nativescript && npm i nativescript-dev-sass --save && cd .. && node tools/install.js",
```

Comment:
just revert this change. no need to modify `postinstall` since `nativescript-dev-sass` is a dependency in nativescript `package.json` now.